examples/seismic/acoustic/wavesolver.py

```diff
@@ -185,11 +214,18 @@ def born(self, dmin, src=None, rec=None, u=None, U=None, m=None, **kwargs):
         Linearized Born modelling function that creates the necessary
         data objects for running an adjoint modelling operator.
 
-        :param src: Symbol with time series data for the injected source term
-        :param rec: Symbol to store interpolated receiver data
-        :param u: (Optional) Symbol to store the computed wavefield
-        :param U: (Optional) Symbol to store the computed wavefield
-        :param m: (Optional) Symbol for the time-constant square slowness
+        Parameters
+        ----------
+        src :
```

Comment:
missing....

Reply:
and the first parameter should ``dmin``

Reply:
@mloubout please provide a description for it

Reply:
```
        src : SparseTimeFunction or array_like, optional
            Time series data for the injected source term.
        rec : SparseTimeFunction or array_like, optional
            The interpolated receiver data.
```
?